docs/guides/connecting_to_your_data/datasource_configuration/how_to_configure_a_pandas_datasource.md

```diff
@@ -0,0 +1,355 @@
+
+For this example, you will be using the `InferredAssetFilesystemDataConnector` as your `class_name`.  This is a subclass of the `InferredAssetDataConnector` that is specialized to support filesystem Execution Engines, such as the `PandasExecutionEngine`.  This key/value entry will therefore look like:
+
+```python
+        "class_name": "InferredAssetFilesystemDataConnector",
+```
+
+For the base directory, you will want to put the relative path of your data from the folder that contains your Data Context.  In this example we will use the same path that was used in the [Getting Started Tutorial, Step 2: Connect to Data](../../../tutorials/getting_started/tutorial_connect_to_data.md).  Since we are manually entering this value rather than letting the CLI generate it, the key/value pair will look like:
+
+```python
+        "base_directory": "./data",
+```
+
+With these values added, along with a blank dictionary for `default_regex` (we will define it in the next step), your full configuration should now look like:
+
+```python
+datasource_config = {
+    "name": "my_datasource_name",
+    "class_name": "Datasource",
+    "module_name": "great_expectations.datasource",
+    "execution_engine": {
+        "class_name": "PandasExecutionEngine",  
+        "module_name": "great_expectations.execution_engine",
+    },
+    "data_connectors": {
+        "name_of_my_inferred_data_connector": {
+            "class_name": "InferredAssetFilesystemDataConnector",
+            "base_directory": "./data",
+            "default_regex": {}
+        }
+    }
+}
+```
+
+  </TabItem>
+  <TabItem value="configured">
+
+  </TabItem>
+  <TabItem value="runtime">
+
+  </TabItem>
+  </Tabs>
+
+
+### 8. Configure your Data Connector's Data Assets
+
+<Tabs
+  groupId="data-asset-type"
+  defaultValue='inferred'
+  values={[
+  {label: 'InferredAssetFilesystemDataConnector', value:'inferred'},
+  {label: 'ConfiguredAssetDataConnector', value:'configured'},
+  {label: 'RuntimeDataConnector', value:'runtime'},
+  ]}>
+
+  <TabItem value="inferred">
+
+
+In an Inferred Asset Data Connector for filesystem data, a regular expression is used to group the files into batches for a Data Asset.  This is done with the value we will define for the Data Connector's `default_regex` key.  The value for this key will consist of a dictionary that contains two values:
+- `pattern`: This is the regex pattern that will define your Data Asset's potential Batch or Batches.
+- `group_names`: This is a list of names that correspond to the groups you defined in `pattern`'s regular expression.
+
+The `pattern` in `default_regex` will be matched against the files in your `base_directory`, and everything that matches against the first group in your regex will become a batch in a Data Asset that possesses the name of the matching text.  Any files that have a matching string for the first group will become Batches in the same Data Asset.
+
```

Comment:
"Batch" or batch? Are they different?

Reply:
Batch, I just missed the capitalization.  I have a script that I run after drafting to catch any words in our Technical Terms glossary that aren't title cased, heh.

Reply:
Fixes for this and one other instance ("batches" instead of "Batches") will be in the next commit.

Reply:
Thanks